crypto/ube/ube_test.cc

```diff
@@ -77,8 +165,8 @@ TEST(Ube, MockedMethodTests) {
   ASSERT_TRUE(CRYPTO_get_ube_generation_number(&generation_number));
   ASSERT_EQ(generation_number, cached_generation_number);
 
-  // Mock another snapsafe event. We used 10 before. Hence, 11 should work.
-  set_snapsafe_generation_number_FOR_TESTING(11);
+  // Mock another fork event but with a strictly smaller value.
+  set_snapsafe_generation_number_FOR_TESTING(5);
```

Comment:
Should this be `set_fork_generation_number_FOR_TESTING`?

Reply:
refactored, redundant now.